### PR TITLE
September news

### PR DESCRIPTION
**Pull request by maelle**

### CODE REVIEW

File: content/blog/2023-09-22-news-september-2023/index.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+     * [GLMMcosinor](https://github.com/ropensci/software-review/issues/603), Fit a cosinor model using a generalised mixed modelling framework. Submitted by [Rex Parsons](https://rwparsons.github.io/). 
+
+    * [weatherOz](https://github.com/ropensci/software-review/issues/598), An API Client for Australian Weather and Climate Data Resources. Submitted by [Rodrigo Pires](https://github.com/bozaah). 
+
+    * [bssm](https://github.com/ropensci/software-review/issues/489), Bayesian Inference of Non-Linear and Non-Gaussian State Space. Submitted by [Jouni Helske](https://jounihelske.netlify.app).  (Stats).
+
+* Two at ['1/editor-checks'](https://github.com/ropensci/software-review/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A1/editor-checks):
+
+     * [agromet](https://github.com/ropensci/software-review/issues/599), Índices y Estadísticos Climáticos e Hidrológicos. Submitted by [Paola Corrales](http://paocorrales.github.io).
```
Comment (steffilazerte) [inline content/blog/2023-09-22-news-september-2023/index.md:145]:
```suggestion
     * [agromet](https://github.com/ropensci/software-review/issues/599), Índices y Estadísticos Climáticos e Hidrológicos. Submitted by [Paola Corrales](https://paocorrales.github.io). 
```

File: content/blog/2023-09-22-news-september-2023/index.Rmd [hunk 2]
```diff
@@ -0,0 +1,553 @@
...
+Dear rOpenSci friends, it's time for our monthly news roundup!
+<!-- blabla -->
+You can read this post [on our blog](`r url`).
+Now let's dive into the activity at and around rOpenSci!
+
+## rOpenSci HQ
+
+### WIP: WebAssembly support in R-universe!
+
+Thanks to some help from George Stagg, we added experimental support for building WebAssembly binary packages. This makes it possible to install packages in [webr](https://docs.r-wasm.org/webr/latest/), directly from r-universe.
```
Comment (steffilazerte) [inline content/blog/2023-09-22-news-september-2023/index.Rmd:45]:
```suggestion
Thanks to some help from George Stagg, we added experimental support for building WebAssembly binary packages. This makes it possible to install packages in [webr](https://docs.r-wasm.org/webr/latest/), directly from the R-universe.
```

File: content/blog/2023-09-22-news-september-2023/index.Rmd [hunk 3]
```diff
@@ -0,0 +1,553 @@
...
+
+For instance: 
+
+```r
+webr::install('jsonlite', repos = c('https://jeroen.r-universe.dev', 'https://repo.r-wasm.org'))
+```
+
+This only works if the package and all of its dependencies support webassembly.
+For CRAN packages you can look at https://repo.r-wasm.org/; for other packages... you have to try.
```
Comment (steffilazerte) [inline content/blog/2023-09-22-news-september-2023/index.Rmd:54]:
```suggestion
For CRAN packages you can look at https://repo.r-wasm.org/; for other packages... you'll have to give it a try!
```

File: content/blog/2023-09-22-news-september-2023/index.Rmd [hunk 4]
```diff
@@ -0,0 +1,553 @@
...
+
+### First champion package to go through peer-review, in Spanish!
```
Comment (steffilazerte) [inline content/blog/2023-09-22-news-september-2023/index.Rmd:56]:
```suggestion
### First rOpenSci Champion's package to go through peer-review, in Spanish!
```

File: content/blog/2023-09-22-news-september-2023/index.Rmd [hunk 5]
```diff
@@ -0,0 +1,553 @@
...
+
+[Carolina Pradier](/blog/2023/05/05/ropensci-champions-program-teams-meet-carolina-pradier-and-athanasia-monika-mowinckel/#carolina-pradier---champion)'s package [eph](https://docs.ropensci.org/eph/) was approved after a [review process in Spanish](https://github.com/ropensci/software-review/issues/593)!
+Congratulations to Carolina, and thanks to editor [Mauro Lepore](/author/mauro-lepore/), reviewers [Guadalupe Gonzalez](https://github.com/guadag12) and [Denisse Fierro Arcos](https://github.com/lidefi87), and mentor [Athanasia Monika Mowinckel](/blog/2023/05/05/ropensci-champions-program-teams-meet-carolina-pradier-and-athanasia-monika-mowinckel/#athanasia-monika-mowinckel---mentor).
+
+eph helps processing data from the Argentina household survey.
```
Comment (steffilazerte) [inline content/blog/2023-09-22-news-september-2023/index.Rmd:61]:
```suggestion
Congratulations to Carolina, and thanks to editor [Mauro Lepore](/author/mauro-lepore/), reviewers [Guadalupe Gonzalez](https://github.com/guadag12) and [Denisse Fierro Arcos](https://github.com/lidefi87), and mentor [Athanasia Monika Mowinckel](/blog/2023/05/05/ropensci-champions-program-teams-meet-carolina-pradier-and-athanasia-monika-mowinckel/#athanasia-monika-mowinckel---mentor). eph is a package which helps process data from the Argentina household survey.
```

File: content/blog/2023-09-22-news-september-2023/index.Rmd [hunk 6]
```diff
@@ -0,0 +1,553 @@
...
+Join us for social coworking & office hours monthly on first Tuesdays! 
+Hosted by Steffi LaZerte and various community hosts. 
+Everyone welcome. 
+No RSVP needed. 
+Consult our [Events](/events) page to find your local time and how to join.
+
+- [Tuesday, October 3rd, 14h00 Europe Central (12:00 UTC)](/events/coworking-2023-10/), Learn about cli and how to make nice command-line interfaces in R! With [community host Athanasia Monika Mowinckel](/author/athanasia-mo-mowinckel/) and [Steffi LaZerte](/author/steffi-lazerte/).
+    - Read up on making nice command line interfaces with [cli](https://cli.r-lib.org/);
+    - Update some scripts or packages to include cleaner and effective interfaces.
+
```
Comment (steffilazerte) [inline content/blog/2023-09-22-news-september-2023/index.Rmd:83]: I will merge in the event PR before Friday so it should be ready by the time you publish 🤞🏼 

```suggestion
- [Tuesday, Nov 7th, 9:00 America Pacific (17:00 UTC)](/events/coworking-2023-11/), Fixing scary bugs 😱🐛 With community host [Salix Dubois](/author/salix-dubois/) and [Steffi LaZerte](/author/steffi-lazerte/).
    - Identify/fix some scary bugs;
    - Chat with Salix about scary bugs!

```

File: content/blog/2023-09-22-news-september-2023/index.Rmd [hunk 7]
```diff
@@ -0,0 +1,553 @@
...
+Join us for social coworking & office hours monthly on first Tuesdays! 
+Hosted by Steffi LaZerte and various community hosts. 
+Everyone welcome. 
+No RSVP needed. 
+Consult our [Events](/events) page to find your local time and how to join.
+
+- [Tuesday, October 3rd, 14h00 Europe Central (12:00 UTC)](/events/coworking-2023-10/), Learn about cli and how to make nice command-line interfaces in R! With [community host Athanasia Monika Mowinckel](/author/athanasia-mo-mowinckel/) and [Steffi LaZerte](/author/steffi-lazerte/).
+    - Read up on making nice command line interfaces with [cli](https://cli.r-lib.org/);
+    - Update some scripts or packages to include cleaner and effective interfaces.
```
Comment (steffilazerte) [inline content/blog/2023-09-22-news-september-2023/index.Rmd:82]:
```suggestion
- [Tuesday, October 3rd, 14h00 Europe Central (12:00 UTC)](/events/coworking-2023-10/), Learn about cli and how to make nice command-line interfaces in R! With community host [Athanasia Monika Mowinckel](/author/athanasia-mo-mowinckel/) and [Steffi LaZerte](/author/steffi-lazerte/).
    - Read up on making nice command line interfaces with [cli](https://cli.r-lib.org/);
    - Update some scripts or packages to include cleaner and effective interfaces;
    - Chat with Mo about using cli
```

File: content/blog/2023-09-22-news-september-2023/index.Rmd [hunk 8]
```diff
@@ -0,0 +1,553 @@
...
+
+### Help needed issues
+
+Don't miss our blog post [Attract Contributors with ‘help wanted’ Issues](/blog/2023/09/19/help-wanted/)! :smile_cat:
+
+### Check your contributing guide...
+
+... with this [evaluation tool](https://contributing.streamlit.app/).
+
+### Regularly check your pkgdown configuration file
```
Comment (steffilazerte) [inline content/blog/2023-09-22-news-september-2023/index.Rmd:532]: Eeep, definitely something I should bear in mind!

**Review by steffilazerte — COMMENTED** [review]
Looks good! I'll confirm that I merged the coworking event PR before Friday so the links are active, thanks!

* [X] post follows Style Guide
* [X] title is in Title Case
* [X] alternative text of images is informative
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on [index.md](http://index.md/)
* [X] I ran `roblog::ro_check_urls()` on [index.md](http://index.md/)
* [X] I ran a spell-check on [index.md](http://index.md/)
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)
* [X] YAML field 'preface' is present if necessary

File: content/blog/2023-09-22-news-september-2023/index.Rmd (hunk 8 above)
Comment (maelle) [reply content/blog/2023-09-22-news-september-2023/index.Rmd:534]: I do that mistake all the time myself :sob:

**Comment by maelle** [comment]
Thank you @steffilazerte! If you don't merge the coworking PR I'll merge it myself tomorrow :grin:

**Comment by steffilazerte** [comment]
I'm waiting on some feedback on the coworking event, so technically I should wait to get that before merging. Hopefully coming in today!

**Pull request merged**